### DISCUSSION TITLE
How to make multiple css files as output?

### DISCUSSION THREAD

**Discussion by kogeletey** [parcel-bundler/lightningcss#668] [Q&A] [answered]
For example, the input contains a file with a design for an article `article` and a file with a design for a header. Each of these files is imported into its own js, for example article.js, and header.js. At the output I want to get two files article.css, and header.css, minified and transformed. How can I do this using vite?

**Comment by devongovett** [comment, marked as answer]
That's a question for the Vite repo, not sure I'll be able to help you here. Lightning CSS only operates on the files you give it. You can choose to bundle them, or transform them and leave the imports in place. How Vite does it is up to them.
